sphinx/tutorial/rose/furthertopics/rose-bunch.rst

```diff
@@ -13,7 +13,8 @@ from each other at the same time - an example would be where a command is
 run repeatedly with only its arguments changing.
 
 Rather than creating multiple apps or
-:ref:`optional configs <rose-tutorial-optional-configurations>` to change
+
+:ref:`optional configs <rose-turorial-optional-configurations>` to change
```

Comment:
Are you deliberately using 'turorial' to link for some reason? I'm a bit confused - obviously just make sure the reference string is consistent everywhere it is used so there are no broken internal reference links.

Reply:
Nevermind, this has been fixed in a future commit.